samples/net/lwm2m_client/src/lwm2m-client.c

```diff
@@ -51,16 +51,8 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
 #define ENDPOINT_LEN		32
 
-#if DT_NODE_HAS_STATUS(DT_ALIAS(led0), okay)
-#define LED_GPIO_PORT	DT_GPIO_LABEL(DT_ALIAS(led0), gpios)
-#define LED_GPIO_PIN	DT_GPIO_PIN(DT_ALIAS(led0), gpios)
-#define LED_GPIO_FLAGS	DT_GPIO_FLAGS(DT_ALIAS(led0), gpios)
-#else
 /* Not an error; the relevant IPSO object will simply not be created. */
```

Comment:
The comment made sense before, now it's a bit awkward. Either remove or modify it.